### PR TITLE
ci(e2e): fix babel compiling error

### PR DESCRIPTION
**Pull request by ShaMan123**
## Motivation

Babel has a silent bug (on windows at least) - it doesn't comiple the files if using babel cli from a spawned call or something like that. It does not throw.
What it means is that when I run e2e tests they will fail ungracefully and I will be the only one to know why.
When I run e2e tests I must copy the cmd from `playwright.setup.ts` and run it with npx for babel to pick it up correctly.

## Description

Move away from using babel cli via a spawn call to **using the API directly**.

### CODE REVIEW

**Review by ShaMan123 — COMMENTED** [review]
DONE
please merge asap

File: playwright.setup.ts [hunk 1]
```diff
@@ -1,16 +1,84 @@
...
+  console.log(
+    `Successfully compiled ${tasks.length} files from ${path.relative(
+      process.cwd(),
+      src
+    )} to ${path.relative(process.cwd(), dist)}`
+  );
+
+  // watch
+  if (process.argv.includes('--ui')) {
+    const watcher = watch(
```
Comment (ShaMan123) [inline playwright.setup.ts:74]: super flaky on windows but good enough to move on

**Comment by asturur** [comment]
if this removes local development headaches i m all for it

Also i think windows is a pain for node development, does it work properly if you run it on the linux subsystem? or is flaky there?

**Review by asturur — APPROVED** [review]

**Comment by ShaMan123** [comment]
I don't know
Seems that windows is a pain full stop

**Pull request merged**